apps/notifications/models.py

```
@@ -0,0 +1,63 @@
+        default=NotificationType.GENERAL,
+    )
+    is_read = models.BooleanField(_('Is read'), default=False)
+    read_at = models.DateTimeField(_('Read at'), null=True, blank=True)
+
+    objects = NotificationQuerySet.as_manager()
+
+    class Meta:
+        verbose_name = _('Notification')
+        verbose_name_plural = _('Notifications')
+        db_table = 'notifications'
+        ordering = ['-created_at']
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)